src/SecureRoute.tsx

```diff
@@ -32,7 +33,9 @@ const SecureRoute: React.FC<{
 
       pendingLogin.current = true;
 
-      oktaAuth.setOriginalUri();
+      const basename = history.createHref({});
+      const currentLocationWithoutBasename = window.location.href.replace(basename, '/');
```

Comment:
Can we get the path for the current route from the router? We can use that path as the originalUri, it should already be relative to the basename

Reply:
it looks like this might not be covering nested routes case